html/manifest/share_target.json

```
@@ -0,0 +1,55 @@
+{
+  "html": {
+    "manifest": {
+      "share_target": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/share_target",
+          "spec_url": "https://w3c.github.io/manifest/#incubations",
+          "support": {
+            "chrome": {
+              "version_added": "89"
+            },
+            "chrome_android": {
+              "version_added": "76"
+            },
+            "edge": {
+              "version_added": "89"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "76"
+            },
+            "opera_android": {
+              "version_added": "63"
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": true
```

```suggestion
              "version_added": "12"
```

This is where they switched to a Chromium engine >76 (https://developer.samsung.com/internet/release-note.html).